README.md

```diff
@@ -1,2 +1,60 @@
-# Challenge
-Challenge
+Objetivo
+======
+
+Dado que o usuário deseja efetuar um caminho por meio do transporte público de Florianópolis, 
+então ele deve encontrar a rota por meio do campo busca, e ao encontrar deverá visualizar a lista
+de ruas, com a possibilidade de visualizar os horários.
+
+Por que Android Annotations?
+========
+
+Diversas facilidades com injeção de view, threads, asynctasks são disponibilidas com AndroidAnnotations, 
+aumentando a produtividade, deixando o código limpo e de coeso.
+
+Por que ActiveAndroid?
+=======
+
+Prefiro usar o ActiveAndroid como ORM pela simplicidade e facilidade na implementação, mas para casos onde
+existe alta performance, prefiro fazer nativamente.
+
+Por que Ion?
+======
+
+Como a camada de acesso a API, é totalmente desacoplada, a justificativa é apenas a rapidez na integração,
```

Comment:
I like Ion because it is more fast for small projects and great for image loading, in my opinion. But, when I need to do complex queries, and it has many endpoints, I prefer to use Retrofit.